src/cr/cube/cubepart.py

```diff
@@ -182,6 +190,35 @@ def inserted_column_idxs(self):
     def inserted_row_idxs(self):
         return tuple(i for i, row in enumerate(self._matrix.rows) if row.is_insertion)
 
+    @lazyproperty
+    def insertions(self):
+        """Returns masked array with residuals for insertions
```

Comment:
oh, this makes more sense now — even compress_rowcols would leave this whole thing intact. this mask is fine, but we may want to provide a way to get just rows or just columns. 🤔 

What do we serialize in second-order for inf, since json is deficient?

Reply:
We use "Infinity" string for `np.inf` ... in the next release we can provide a way to provide only rows, only cols or both; also we can change the mask using the compress thing of numpy